docs/releases.rst

```diff
@@ -5,6 +5,15 @@
 ======================
 
 
+tmt-1.44.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Execute tmt option ``--ignore-duration`` makes tmt to execute
+the test as long as it needs. This can be enabled via environment
+``TMT_PLUGIN_EXECUTE_TMT_IGNORE_DURATION=1`` even when the execute
+step is not stated on the commandline.
```

Comment:
* I'd recommend sticking to "key" rather than "option", because fmf is meant to be the primary source (https://tmt.readthedocs.io/en/stable/contribute.html#docs)
* I'd not include the note about envvar, on the basis that every plugin and step option has this feature. It feels like an exceptional granularity to me.

Reply:
> I'd not include the note about envvar, on the basis that every plugin and step option has this feature

I want to make clear that it will work even when plugin is not mentioned on CLI, which in this case is the majority of executions. Otherwise, if one knows this limitation, they will either modify their plans or avoid using option in TF

Reply:
Reworded in field message and add note into overview section in 2051375e